packages/app/src/systems/Bridge/components/NetworkConnectCard/NetworkConnectCard.tsx

```diff
@@ -0,0 +1,86 @@
+import { cssObj } from '@fuel-ui/css';
+import { Card, Stack, Text, Flex, Image, Button, Icon } from '@fuel-ui/react';
+import type { Bech32Address } from 'fuels';
+
+import { formatAddress } from '~/systems/Core/utils';
+
+type NetworkConnectCardProps = {
+  networkName: string;
+  networkImageUrl: string;
+  heading: string;
+  isConnecting: boolean;
+  currentAccount?: `0x${string}` | Bech32Address;
+  accounts?: `0x${string}`[] | Bech32Address[];
+  onConnect: () => void;
+};
+
+export const NetworkConnectCard = ({
+  networkName,
+  networkImageUrl,
+  heading,
+  isConnecting,
+  currentAccount,
+  onConnect,
+}: NetworkConnectCardProps) => {
+  return (
+    <Card css={styles.root}>
+      <Card.Body css={styles.body}>
+        <Flex justify="space-between">
+          <Stack gap="$1">
+            <Text fontSize="sm">{heading}</Text>
+            <Flex gap="$2" align="center">
+              <Image width="20" height="20" src={networkImageUrl} />
+              <Text color="blackA12">{networkName}</Text>
+            </Flex>
+          </Stack>
+          <Flex align="end">
+            {!currentAccount ? (
+              <Button
+                onPress={onConnect}
+                isLoading={isConnecting}
+                css={styles.buttonRoot}
+              >
+                Connect wallet
+              </Button>
+            ) : (
+              <Button
+                onPress={onConnect}
+                isLoading={isConnecting}
+                leftIcon={<Icon icon="Circle" color="red10" fill="red10" />}
+                rightIcon="CaretDown"
+                variant="outlined"
+                css={{ ...styles.buttonRoot, ...styles.accountButton }}
+              >
+                {formatAddress(currentAccount)}
+              </Button>
+            )}
```

Comment:
In order to align the center button and network, we need to change the structure here to something like this;
```suggestion
          </Stack>
          <Flex align="space-between" justify="center">
             <Flex gap="$2" align="center">
              <Image width="20" height="20" src={networkImageUrl} />
              <Text color="blackA12">{networkName}</Text>
            </Flex>
            <Box>
            {!currentAccount ? (
              <Button
                onPress={onConnect}
                isLoading={isConnecting}
                css={styles.buttonRoot}
              >
                Connect wallet
              </Button>
            ) : (
              <Button
                onPress={onConnect}
                isLoading={isConnecting}
                leftIcon={<Icon icon="Circle" color="red10" fill="red10" />}
                rightIcon="CaretDown"
                variant="outlined"
                css={{ ...styles.buttonRoot, ...styles.accountButton }}
              >
                {formatAddress(currentAccount)}
              </Button>
            )}
            <Box>
          </Flex>
```